Source/player.cpp

```diff
@@ -2699,7 +2699,7 @@ void AddPlrExperience(Player &player, int lvl, int exp)
 	uint32_t clampedExp = std::max(static_cast<int>(exp * (1 + (lvl - player._pLevel) / 10.0)), 0);
 
 	// Prevent power leveling
-	if (gbIsMultiplayer) {
+	if (gbIsMultiplayer && !debug) {
 		const uint32_t clampedPlayerLevel = clamp(static_cast<int>(player._pLevel), 1, MaxCharacterLevel);

```

Comment:
```suggestion
#ifndef debug
	if (gbIsMultiplayer) {
		const uint32_t clampedPlayerLevel = clamp(static_cast<int>(player._pLevel), 1, MaxCharacterLevel);
#endif
```

Reply:
It would

Reply:
> It would
> 
> 

Maybe we can revisit this command if someone else has a need for it specifically that they can't use the command thats already there for